tox.ini

```diff
@@ -9,25 +11,44 @@ envlist =
     coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
 extras = test
-deps = zope.testrunner
+
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+commands =
+    flake8 src setup.py
+    check-manifest
 
 [testenv:coverage]
-basepython =
-    python3
+basepython = python3
+deps =
+    coverage
+    zope.testrunner
 commands =
     coverage run -m zope.testrunner --test-path=src []
     coverage html
-    coverage report --fail-under=98
-deps =
-    zope.testrunner
-    coverage
+    coverage report -m
 
-[testenv:flake8]
-basepython =
-    python3
-skip_install = true
-deps = flake8
-commands = flake8 --doctests src setup.py
+[coverage:run]
+branch = True
+source = src
```

Comment:
`.coveragerc` had `source = z3c.layer.pagelet`, which in theory would allow us to track coverage even without `usedevelop = true`.